test/WebJobs.Script.Tests/Security/SecretManagerTests.cs

```diff
@@ -496,9 +496,9 @@ public async Task GetHostSecrets_WhenNonDecryptedHostSecrets_SavesAndRefreshes()
                 }
 
                 Assert.NotNull(hostSecrets);
-                Assert.Equal(hostSecrets.MasterKey, "cryptoError");
+                Assert.NotEqual(hostSecrets.MasterKey, "cryptoError");
```

Comment:
Were the tests asserting invalid behavior before?

Reply:
Yes, the key should be new in this case.